### PR TITLE
docs(forms): correct sample code for FormArray.reset

### PR DESCRIPTION
**Pull request by bethge**
Remove unexpected `this`, correct output comment for `arr.value` and correct parameter type for `FormArray.get()`.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/forms/src/model.ts [hunk 1]
```diff
@@ -2091,19 +2091,19 @@ export class FormArray extends AbstractControl {
...
-   * this.arr.reset([
+   * arr.reset([
    *   {value: 'name', disabled: true},
    *   'last'
    * ]);
    *
-   * console.log(this.arr.value);  // ['name', 'last name']
-   * console.log(this.arr.get(0).status);  // 'DISABLED'
+   * console.log(arr.value);  // ['last']
+   * console.log(arr.get('0').status);  // 'DISABLED'
```
Comment (AndrewKushnir) [inline packages/forms/src/model.ts:2106]: I think we can use `at` here:
```suggestion
   * console.log(arr.at(0).status);  // 'DISABLED'
```

**Review by AndrewKushnir — COMMENTED** [review]
@bethge the change looks good, thanks for improving Angular docs! 👍 

I've just left a comment to use `at` instead of `get`. Could you please have a look when you get a chance?

File: packages/forms/src/model.ts (hunk 1 above)
Comment (AndrewKushnir) [reply packages/forms/src/model.ts]: FYI since it was a minor update I've applied this change and marked this PR as ready for merge.

**Review by AndrewKushnir — APPROVED** [review]

**Comment by bethge** [comment]
Sorry for the delay, Andrew - I lost track of this one. Thank you for applying the patch directly and merging.

**Pull request closed without merging**